stellar/deploy-contract.js

```diff
@@ -8,12 +8,77 @@ const { getDomainSeparator, getChainConfig } = require('../common');
 const { prompt, validateParameters } = require('../common/utils');
 const { weightedSignersToScVal } = require('./type-utils');
 const { ethers } = require('hardhat');
-const { readFileSync } = require('fs');
+const { writeFileSync, readFileSync } = require('fs');
+const fetch = require('node-fetch');
+const path = require('path');
 const {
     utils: { arrayify, id },
 } = ethers;
 require('./cli-utils');
 
+const AXELAR_RELEASE_BASE_URL = 'https://static.axelar.network/releases/axelar-cgp-stellar';
```

Comment:
Update the release action to use `releases/stellar` as the base path and move the existing folder on R2. The repo name shouldn't matter (and thus shouldn't cause it to break if we ever change it again)